lib/wisper/activerecord/publisher.rb

```diff
@@ -45,6 +45,9 @@ def broadcast_event_name(lifecycle)
     end
   end
 end
-ActiveSupport.on_load(:active_record) do
-  include Wisper::ActiveRecord::Publisher
+
+ActiveSupport::Deprecation.silence do 
```

Comment:
good find!  I'm on board with adding this call, but I'd like to only wrap the offending statements instead of the entire on_load block.  there may be other relevant deprecation warnings that are swallowed by wrapping the entire block.

I believe wrapping just the `after_commit` calls should be sufficient.